modules/theme/src/definitions/globals/product.less

```diff
@@ -73,6 +75,140 @@ body {
             width: 100px !important;
         }
     }
+
+    .icon-wrapper {
+        background-color: @iconWrapperBgColor;
+        overflow: hidden;
+        display: flex;
+        text-align: center;
+        align-items: center;
+        justify-content: center;
+        &.inline {
+            display: inline-block;
+        }
+        &.transparent {
+            background-color: transparent;
+        }
+        &.relaxed {
+            padding: @iconWrapperPadding;
+            &.very {
+                padding: @iconWrapperVeryPadding;
+            }
+        }
+        &.bordered {
+            border-width: @iconWrapperBorderWidth;
+            border-style: solid;
+            border-color: @iconWrapperBorderColor;
+        }
+        &.rounded {
+            border-radius: @defaultBorderRadius;
+        }
+        &.default {
+            svg.icon {
+                path {
+                    fill: @iconDefaultBgColor;
+                }
+            }
+        }
+        &.mini {
+            svg.icon {
+                width: 35px;
```

Comment:
Move sizes to a variable in `global.variables` file and use

Reply:
Sizes have been moved to ```product.variables``` file.